lib/fail.helper.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+if (!jasmine.fail) {
+    /**
+     * Sets up the scenario to actually fail; `actual` and `expected`
+     * need to never match.
+     *
+     * @param {string} actual
+     * @param {string} expected
+     * @param {Function} used just for testing purposes
```

Comment:
The variable name `expectFn` is not on this line.  eslint should have barked loudly.